docs/quickstart.rst

```diff
@@ -0,0 +1,43 @@
+==============================
+Getting started with allofplos
+==============================
+
+allofplos comes with a "starter corpus" so that right after install, you can begin to analyze a set of 122 PLOS articles using the Corpus and Article classes.
+
+    >>> from allofplos import Corpus, Article, starterdir
```

Comment:
I would remove the discussion of the Corpus class for now, because we can move this into the main repo as long as you aren't using the Corpus class.